firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -54,6 +54,22 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
         // Swipe to the Homescreen
         app.buttons["\(rootA11yId)SecondaryButton"].tap()
         currentScreen += 1
+        mozWaitForElementToExist(app.scrollViews.staticTexts["\(rootA11yId)TitleLabel"])
+        mozWaitForElementToExist(app.scrollViews.staticTexts["\(rootA11yId)DescriptionLabel"])
+        mozWaitForElementToExist(app.buttons["\(rootA11yId)PrimaryButton"])
+        mozWaitForElementToExist(app.buttons["\(rootA11yId)SecondaryButton"])
+        snapshot("Onboarding-4")
```

Comment:
Here are the new screens:
![Simulator Screenshot - iPhone 15 - 2024-09-03 at 23 49 48](https://github.com/user-attachments/assets/477e1764-2058-4c7f-aa33-79308ed56d0b)
![Simulator Screenshot - iPhone 15 - 2024-09-04 at 00 22 24](https://github.com/user-attachments/assets/dfc11620-b7a5-443c-b205-89cc744a5a3f)